legal-api/report-templates/template-parts/business-summary/stateTransition.html

```diff
@@ -35,6 +35,9 @@
                 <div class="pt-2">
                   <span>Incorporation Number: {{ filing.identifier }}</span>
                 </div>
+              {% elif filing.filingType == 'dissolution' and filing.filingName == 'Involuntary Dissolution' %}
```

Comment:
I think maybe you should be using `filing.filngSubType == involutnary` instead of `filing.filingName == Involuntary Dissolution`.

Reply:
Thanks, Argus.

I have updated the condition to use filing.filingSubType.

Reply:
were you able to confirm you could pull back the business summary properly with this update?

Reply:
Yes, fixed the issue and confirmed the latest business summary is displaying correctly with this update. 
Thanks again.